afl/build.rs

```diff
@@ -16,11 +16,25 @@ mod common;
 fn main() {
     let installing = home::cargo_home()
         .map(|path| Path::new(env!("CARGO_MANIFEST_DIR")).starts_with(path))
-        .unwrap();
+        .unwrap()
+        || env::var("TESTING_INSTALL").is_ok();
+
+    let building_cargo_afl = env::var("CARGO_PKG_NAME") == Ok(String::from("cargo-afl"));
+
+    let building_on_docs_rs = env::var("DOCS_RS").is_ok();
+
     let out_dir = env::var("OUT_DIR").unwrap();
 
+    if installing && !building_cargo_afl {
+        println!("cargo:warning=You appear to be installing `cargo-afl` with:");
+        println!("cargo:warning=    cargo install afl");
+        println!("cargo:warning=A future version of afl.rs will require you to use:");
+        println!("cargo:warning=    cargo install cargo-afl");
+        println!("cargo:warning=You can use the new command now, if you like.");
+    }
```

Comment:
Does it make sense to also suggest `cargo uninstall afl`, or is the latter `cargo-afl` going to effectively clobber the install anyways?

Reply:
Rather than suggest to uninstall, I propose the following. Change the first line to:
```
warning: You appear to be installing the `cargo-afl` binary with:
```
And add the following line at the end:
```
warning: Note: If the binary is already installed, you may need to add --force.
```
So the revised message would look like:
```
warning: You appear to be installing the `cargo-afl` binary with:
warning:     cargo install afl
warning: A future version of afl.rs will require you to use:
warning:     cargo install cargo-afl
warning: You can use the new command now, if you like.
warning: Note: If the binary is already installed, you may need to add --force.
```
What do you think?

Reply:
Sounds good to me!